h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -67,6 +68,18 @@ describe('CreateEditGroupForm', () => {
     return Boolean(component.prop('error'));
   };
 
+  const getSelectedGroupType = wrapper => {
+    return wrapper.find('[data-testid="group-type"]').prop('selected');
+  };
+
+  const setSelectedGroupType = (wrapper, newType) => {
+    const radioGroup = wrapper.find('[data-testid="group-type"]');
+    act(() => {
+      radioGroup.prop('onChange')(newType);
+    });
```

Comment:
Using `act` here ensures that the form is immediately re-rendered after we change the state by toggling the radio buttons.